### PR TITLE
imjp19

### PR DESCRIPTION
**Pull request by imjp19**
- [X] I have read the Code Of Conduct.
- [X] I have followed all the steps of submission properly.

### CODE REVIEW

**Comment by imjp19** [comment]
Now please check once.

**Comment by jayskhatri** [comment]
Thanks @imjp19 , for opening the pull request! 👏

I'll be reviewing your pull request thoroughly

Star ⭐ this project and [tweet](https://twitter.com/share?text=Contributed+to+awesome+projects+at+@club_gamma&url=https://clubgamma.github.io/hacktoberfest) about your contributions and also you can join the [discord channel of club gamma](https://discord.gg/kjnp6wU)

File: README.md [hunk 1]
```diff
@@ -51,22 +51,11 @@ Before start contributing please read the [CONTRIBUTING.md](https://github.com/c
...
-| <img src="https://github.com/HetGalia/Awesome-Web-Art/blob/master/HetGalia/Pendulum/Pendulum.gif" width=80% height=70%> | [Find the code here](https://github.com/HetGalia/Awesome-Web-Art/tree/master/HetGalia/Pendulum) | [HetGalia](https://github.com/HetGalia) |
-| <img src="https://github.com/kashyap-shah/Awesome-Web-Art/blob/master/kashyap-shah/Weather_App.gif" width=80% height=70%> | [Find the code here](https://github.com/kashyap-shah/Awesome-Web-Art/tree/master/kashyap-shah/Weather%20App%20-%20Javascript%20%26%20API) | [Kashyap Shah](https://github.com/kashyap-shah) |
-| <img src="https://github.com/HetGalia/Awesome-Web-Art/blob/master/HetGalia/tic-tac-toe/Tic%20Tac%20Toe.gif" width=80% height=70%> | [Find the code here](https://github.com/HetGalia/Awesome-Web-Art/tree/master/HetGalia/tic-tac-toe) | [HetGalia](https://github.com/HetGalia) |
+
+
+| <img src="https://github.com/imjp19/Awesome-Web-Art/blob/master/imjp19/HOVER.gif" width=80% height=70%> | [Find the code here](https://github.com/imjp19/Awesome-Web-Art/blob/master/imjp19/button-hover.html) | [Jay Patel](https://github.com/imjp19) |
```
Comment (jayskhatri) [inline README.md:54]: Hey @imjp19 what you have added is not in proper aligned table, [check here](https://github.com/clubgamma/Awesome-Web-Art/blob/021a370bcb4fc894b174919031c25f9a8f4b606a/README.md) in the last 2 rows the table is broke.

**Review by jayskhatri — CHANGES_REQUESTED** [review]
Please do necessary changes then after PR is ready to be merged

Happy hacking🚀

**Review by jayskhatri — APPROVED** [review]
Good to go👍

**Comment by yashpaneliya** [comment]
Hello contributor👋,

As we all know that Hacktoberfest-2020 is ended. Hope you enjoyed it a lot and made some awesome contributions. 
So now it's time for schwags🎁🤩

**Send your email (linked with your GitHub account) in the reply of this comment. Mention any one of the maintainers (@yashpaneliya @rudrabarad @harshptl14 @chirayu-joshi @dhruvpatel1706 ) in the reply.**

**We'll send a form on the email you'll provide. Please reply ASAP.**

Join [Club Gamma Discord](https://discord.com/invite/kjnp6wU) for future updates.

**Comment by imjp19** [comment]
jaypatel19124@gmail.com @yashpaneliya

**Pull request merged**